integrationtest/src/de/tobiasroeser/mill/integrationtest/MillIntegrationTestModule.scala

```diff
@@ -235,14 +235,18 @@ trait MillIntegrationTestModule extends TaskModule {
     }
     val url = s"https://github.com/lihaoyi/mill/releases/download/${mainVersion.mkString(".")}/$v$suffix"
 
+    // download mill to to a same path as in this script: https://github.com/lihaoyi/mill/blob/master/mill
+    val cacheDir = T.env
+      .get("XDG_CACHE_HOME")
+      .map(os.Path(_))
+      .getOrElse(os.home / ".cache")
+    val target = cacheDir / "mill" / "download" / v
```

Comment:
I've considered this one too, but it would be great if we make it configurable it to also support the download to `out` dir.

Reply:
I have made this behavior configurable